OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -492,8 +492,10 @@ addPathPoint(const SimTK::State& s, int aIndex, const PhysicalFrame& frame)
 {
     PathPoint* newPoint = new PathPoint();
     newPoint->setParentFrame(frame);
-    Vec3 location = newPoint->getLocation(s);
+    Vec3 location = newPoint->getLocation(s); // 0 by construction
+    // location is now a COPY not a reference so the computed location is LOST unless setLocation is invoked
```

Comment:
Also do not understand the relevance of  this comment. `location` is a `Vec3` above not a `Vec3&`!  What is lost?
The passing  in of a reference you expect to get overwritten in `placeNewPathPoint` is what is unexpected/wrong in this scheme and that should be documented here.

Reply:
@aseth1 The comments were for myself trying to understand what went wrong when the change was made to getLocation(s). I agree the current call sequence doesn't make sense, I should refer you to the developer who made the change in the first place 😃 

I'll update anyway per your feedback. Thank you.